website/docs/docs/deploy/job-scheduler.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "The dbt Cloud job scheduler"
+id: "job-scheduler"
+sidebar_label: "Job scheduler"
+description: "The dbt Cloud Scheduler queues scheduled or API-triggered runs, before preparing the job to enter cloud data platform. Build observability into transformation workflows with the in-app scheduling, logging, and alerting." 
+tags: [scheduler]
+---
+
+The dbt Cloud job scheduler is a powerful tool that helps data teams optimize their data transformation workflows, makes it easier to run dbt in production environments, governs data, automates your dbt jobs, and empowers teams to get results faster with less effort. 
```

Comment:
```suggestion
The dbt Cloud job scheduler is a powerful tool that helps data teams optimize their data transformation workflows, makes it easier to run dbt in production environments, governs data, automates your dbt jobs, and empowers teams to get results faster with less effort. 
```
The Scheduler is the backbone of running jobs in dbt Cloud, bringing power and simplicity to building data pipelines in both continuous integration and production contexts. The Scheduler frees teams from having to build and maintain their own infrastructure and ensures the timeliness and reliability of data transformations.

The Scheduler enables both cron-based and event-driven execution of dbt commands in the user’s warehouse. Specifically, the Scheduler handles:

- Cron-based execution of dbt Cloud jobs that run on a predetermined cadence
- Event-driven execution of dbt Cloud CI jobs triggered by pull requests to the the dbt repo
- Event-driven execution of dbt Cloud jobs triggered by API

The Scheduler takes care of:

- queuing jobs
- creating short-lived environments to execute jobs’ dbt run commands
- delivering logs which are needed for debugging and remediation
- storing dbt artifacts for direct consumption and ingestion by the Metadata API

The Scheduler powers running dbt in staging and production environments, bringing ease and confidence to CI/CD workflows and enabling observability and governance in deploying dbt at scale.